api/constants/constants.go

```diff
@@ -174,3 +174,8 @@ const (
 	// ALPNSNIProtocolReverseTunnel is TLS ALPN protocol value used to indicate Proxy reversetunnel protocol.
 	ALPNSNIProtocolReverseTunnel = "teleport-reversetunnel"
 )
+
+const (
+	// KubeSNIPrefix is a SNI Kubernetes prefix used for distinguishing the Kubernetes HTTP traffic.
+	KubeSNIPrefix = "kube."
```

Comment:
@smallinsky I'm thinking that `kube.` prefix might be a little too generic still as I can imagine customers using it e.g. for Kubernetes dashboard behind app access, etc. Can we pick something more specific that users will be less likely to choose for their public address? For example, `kube-teleport-proxy-alpn.` or something like that.

Reply:
@r0mant  Yes, the `kube-teleport-proxy-alpn.` sound reasonable though changing the prefix will require  users to call `tsh kube login` to regenerate the ServerName in k8 user cert. Also the tsh version on the client side needs to be updated to a new server name prefix in kube config. So I will make support to `kube.` and  a new `kube-teleport-proxy-alpn.` in v8 release where in master the `kube-teleport-proxy-alpn.` can be only supported.

Reply:
Sounds good to me.

Reply:
Updated.